src/special/serverless.js

```diff
@@ -0,0 +1,32 @@
+import * as path from 'path';
+import yaml from 'js-yaml';
+import { getContent } from '../utils/file';
+
+/**
+ * Get plugin names from a yaml object.
+ * @param {*} yml parsed serverless configuration
+ */
+function getPlugins(serverlessConfig) {
+  return serverlessConfig.plugins;
+}
+
+/**
+ * Get the dependency names of the given plugins.
+ * @param {*} plugins array of plugin names as strings
+ */
+function getDependencies(plugins) {
+  return plugins;
+}
+
+export default async function parseServerless(filename) {
+  const basename = path.basename(filename);
+
+  if (basename === 'serverless.yml') {
+    const content = await getContent(filename);
+    const config = yaml.safeLoad(content);
+    // TODO find plugins defined in included files
```

Comment:
This todo sems like it's been done no?

Reply:
This detects only plugins from the main serverless.yml, but I think you could have plugins in included files like this:
```
plugins: ${file(./serverless.plugins.yml)}
```

Reply:
Oooh, ok great, could you add that comment in the TODO so that we know what is not implemented?

Reply:
Done 👍